tests/metrics/time/README.md

```diff
@@ -0,0 +1,19 @@
+# Kata Containers boot time metrics
```

Comment:
@dborquez I think that in order to add this README first we need this one https://github.com/kata-containers/kata-containers/pull/7158 to be merged in order to make a reference of this file as if we do not do it the static checks will fail

Reply:
@dborquez maybe we can submit this one later